lib/iex/lib/iex/dochelp/erlang_stub.ex

```diff
@@ -0,0 +1,48 @@
+defmodule IEx.DocHelp.ErlangStub do
+  @moduledoc """
+  Stub module to suggest possible addons for Erlang documentation.
+
+  Currently only returns {:unknown, doc_list }
+  with a suggestive error message.
+  """
+
+  @suggestive "and currently there is no helper installed to provide Erlang documentation"
```

Comment:
This message makes it appear as if you are missing something that needs to be installed (when I don't believe there is anything that can be installed to remedy this).